testing/kuttl/e2e/pgbouncer/00-cluster.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: pgbouncer-test
+spec:
+  postgresVersion: 14
+  instances:
+    - name: instance1
+      replicas: 2
+      dataVolumeClaimSpec:
+        accessModes:
+        - "ReadWriteOnce"
+        resources:
+          requests:
+            storage: 1Gi
+  backups:
+    pgbackrest:
+      repos:
+      - name: repo1
+        volume:
+          volumeClaimSpec:
+            accessModes:
+            - "ReadWriteOnce"
+            resources:
+              requests:
+                storage: 1Gi
+  proxy:
+    pgBouncer:
+      image: registry.developers.crunchydata.com/crunchydata/crunchy-pgbouncer:centos8-1.16-0
+      replicas: 1
+      affinity:
```

Comment:
🤔 Why specify affinity when `replicas: 1` ? Is there supposed to be an assertion about it?

Reply:
No, just copy-paste from the -examples repo (except I tuned the replicas down to `1` -- I could bump it back up, but the pattern we're setting seems to be around focused tests, i.e., we need a test just for affinity...)